examples/source/1.components/amp-analytics/embed.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style media="screen" type="text/css">
+      body {
+        background: white;
+        font-family: Roboto,Helvetica,Arial,sans-serif;
```

Comment:
We're no longer using Robots on the site. Please replace it with our current fonts.

Reply:
Thanks, PTAL.